app/src/main/assets/drug_stock_no_connection.html

```diff
@@ -0,0 +1,8 @@
+<html>
+<body style="background: #F0F2F5; font-family: Roboto, sans-serif; text-align: center;">
+<div style="font-size: 24px; font-weight: bold; padding: 24px 32px 8px 32px; color: #2F363D;">Not available</div>
+<div style="padding: 0 32px 24px 32px; color: #6C737A;">An internet connection is needed to use this feature.</div>
```

Comment:
These user visible strings need to be localised, no?

Reply:
Ideally yes, but I am unsure how translations would work for HTML files. Is there a specific format we should follow here? 🤔

Reply:
Yeah. Let's go with a later option, we can add it now and improve it later if someone complains regarding the translations. For now, doing it during runtime seems like a bit of work to implement 😅